js/src/sdk/client/schemas.gen.ts

```diff
@@ -2672,6 +2689,21 @@ export const $GetTriggerResponseDTO = {
     required: ['status']
 } as const;
 
+export const $WehbookNewFormatDTO = {
```

Comment:
There's a typo in the type name: `WehbookNewFormatDTO` should be `WebhookNewFormatDTO`. This could cause confusion and inconsistency in the codebase.